profiles/tests/test_filter_profiles.py

```diff
@@ -129,14 +129,14 @@ def test_get_all_profiles_authorized_filter_activity_retail(self):
                        3 in response.data[i]["comp_activity"]]
         self.assertEqual(len(retail_type), filtered_len)
 
-    def test_get_all_profiles_authorized_filter_activity_HORACE(self):
-        response = self.client.get(f"/api/profiles/?activity_type=HORACE")
+    def test_get_all_profiles_authorized_filter_activity_HORECA(self):
```

Comment:
In method names it _definitely_ must be lowercase.

Reply:
Done.